lib/jbuild

```diff
@@ -1,33 +1,9 @@
-(* -*- tuareg -*- *)
-#require "unix"
-
-let flags = function
-| [] -> ""
-| pkgs ->
-  let cmd = "ocamlfind ocamlc -verbose" ^ (
-    List.fold_left (fun acc pkg -> acc ^ " -package " ^ pkg) "" pkgs
-  ) in
-  let ic = Unix.open_process_in
-    (cmd ^ " | grep -oEe '-ppx (\"([^\"\\]|\\.)+\"|\\w+)'")
-  in
-  let rec go ic acc =
-    try go ic (acc ^ " " ^ input_line ic) with End_of_file -> close_in ic; acc
-  in
-  go ic ""
-
-let rewriters = ["ppx_deriving_rpc"; "ppx_sexp_conv"]
-
-let () = Printf.ksprintf Jbuild_plugin.V1.send {|
 (jbuild_version 1)
 
 (library
  ((name backtrace)
   (public_name xapi-backtrace)
-  (libraries (ppx_sexp_conv.runtime-lib
-              rpclib
-              rpclib.json
-              threads
-              sexplib))
-  (flags (:standard %s))
- ))
-|} (flags rewriters)
+  (flags (:standard -w -39-32))
+  (libraries
+   (ppx_sexp_conv.runtime-lib rpclib rpclib.json threads sexplib))
```

Comment:
Is sexplib needed? It should be in principle replaceable by `Sexplib0` in `ppx_sexp_conv.runtime-lib`

Reply:
Trying to compile without `sexplib` fails. So I assume it is required but I also find this a bit strange.

Reply:
The [documentation](https://github.com/janestreet/ppx_sexp_conv)  says explicitly:

```
These sexp-converters depend on having a set of converters for basic values (e.g., int_of_sexp) already in scope. This can be done by writing:

open Sexplib.Std
```
I think that this is a bad design on the part of the generated code but there isn't much we can do about this here.

Reply:
ppx_type_conv runtime libs reexports sexplib0 that includes these functions and is self-standing (does not require base afaik): https://github.com/janestreet/sexplib0/blob/master/sexp_conv.ml

Reply:
I have updated to PR to remove this dependency.